src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/Statistics/BoundaryLevelViewModel.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels.Statistics;
+
+public class BoundaryLevelViewModel
+{
+    public long Id { get; set; }
+
+    public string Level { get; set; }
+
+    public string Label { get; set; }
+
+    public DateTime Published { get; set; }
```

Comment:
Similar comments to those I left on `BoundaryLevelUpdateRequest`, i.e. you could use `record` instead of `class` here, you could use the `required` keyword and all the `set` properties could be replaced with `init`.

It would look a bit like this

```csharp
public record BoundaryLevelViewModel
{
    public required long Id { get; init; }

    public required GeographicLevel Level { get; init; }

    public required string Label { get; init; }

    public required DateTime Published { get; init; }
}
```